frontend/views/shared/_subrecord_form.html.erb

```diff
@@ -0,0 +1,55 @@
+<%
+  plugin = false if plugin.blank?
+  i18n_prefix = plugin ? 'plugins.' : ''
+  singular = name.singularize
+  section_id = form.id_for(name) if section_id.blank?
+  heading_size = "h3" if heading_size.blank?
+  heading_text = I18n.t("#{i18n_prefix}#{singular}._plural") if heading_text.blank?
+  template = singular if template.blank?
+  template_erb = "#{name.to_s}/template" if template_erb.blank?
+  js_template_name = "template_#{template}" if js_template_name.blank?
+  custom_action_template = false if custom_action_template.blank?
+  help_topic = form.help_path_for(name) if help_topic.blank?
+  really_hidden = false if really_hidden.blank?
+  hidden = really_hidden
+  section_class = hidden ? "subrecord-form-hidden" : "subrecord-form"
+
+  required = form.required?(name) if required.blank?
+
+  cardinality = :zero_to_many if cardinality.blank?
+%>
+
+<%= render_aspace_partial :partial => template_erb, :locals => {:form => form} %>
+
+
+<section id="<%= section_id %>" class="<%= section_class %><% if required == :conditionally %> conditionally-required<% elsif required %> required<% end %>" data-subrecord-form="true" data-object-name="<%= singular %>" data-template="<%= js_template_name %>" <% if custom_action_template %>data-custom-action="true"<% end %> data-cardinality="<%= cardinality %>">
+  <% unless hidden %>
+  <<%= heading_size %> class="subrecord-form-heading">
+    <%= wrap_with_tooltip(heading_text, "#{form.i18n_for(name)}_tooltip", "subrecord-form-heading-label") %>
+
+    <% if !form.readonly? %>
+     <%= link_to_help :topic => help_topic %>
+     <% if custom_action_template %>
+       <%= render :partial => custom_action_template %>
+     <% else %>
+       <button class="btn btn-sm btn-default pull-right show-all" style="display: none;"><%= I18n.t("#{i18n_prefix}.actions.show_all") %></button>
+       <button class="btn btn-sm btn-default pull-right"><%= I18n.t("#{i18n_prefix}#{singular}._frontend.action.add") %></button>
+     <% end %>
+    <% end %>
+
+  </<%= heading_size %>>
+  <% end %>
+
+  <div class="subrecord-form-container">
+    <% if cardinality === :zero_to_one %>
+      <%= form.list_for([form[name]].compact, name) do |item| %>
+        <% form.emit_template(template, item) %>
+      <% end %>
+    <% else %>
+      <%= form.list_for(form[name], "#{name}[]") do |item| %>
+        <% form.emit_template(template, item) %>
+      <% end %>
+    <% end %>
+  </div>
+
+</section>
```

Comment:
Sorry.. might be missing something again! Is there a plugin customisation in there?